### PR TITLE
[TACHYON-1376] new test case for getUsedBytes command

### PR DESCRIPTION
**Pull request by CruncherBigData**
part of my efforts in covering items for JIRA https://tachyon.atlassian.net/browse/TACHYON-1376

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import tachyon.client.TachyonFSTestUtils;
+import tachyon.client.TachyonStorageType;
+import tachyon.client.UnderStorageType;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.block.TachyonBlockStore;
+import tachyon.shell.AbstractTfsShellTest;
+import tachyon.TachyonURI;
+
+/**
+ * Tests the "getUsedBytes" command
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: "getUsedBytes" -> {@link GetUsedBytesCommand}

Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: period at end of first sentence of javadoc

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class GetUsedBytesCommandTest extends AbstractTfsShellTest{
+  /**
+   * Tests the available memory on the cluster after adding couple of files
+   */
+  @Test
+  public void getUsedBytesTest() throws Exception {
+    String filePathA = "/testFileA";
+    String filePathB = "/testFileB";
+    long tachyonCapacity = TachyonBlockStore.get().getCapacityBytes();
+    int fileASize = (int)(0.2 * (double)tachyonCapacity);
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: casts to `(double)` are unnecessary

File: integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java [hunk 3]
```diff
@@ -89,7 +89,7 @@ public void setPinTest() throws Exception {
     int fileAInMemoryPercentage = mTfs.getInfo(fileA).inMemoryPercentage;
     int fileBInMemoryPercentage = mTfs.getInfo(fileB).inMemoryPercentage;
     Assert.assertEquals(100, fileAInMemoryPercentage);
-    Assert.assertTrue("inMemoryPercentage for fileB  should be less than (" + fileBSize + ")",
+    Assert.assertTrue("inMemoryPercentage for fileB  should be less than 100%",
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java]: This change should be in a different PR

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    int fileBSize = (int)(0.3 * (double)tachyonCapacity);
+
+    // Ensures initial used memory is zero
+    Assert.assertEquals(0, TachyonBlockStore.get().getUsedBytes());
+
+    //Add first file with 5MB to consume 50% 0f existing memory size
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: 0f -> of

Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: space before Add

Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: Remove "with 5MB" - this test doesn't need to know the default cluster capacity

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    TachyonFile fileA = TachyonFSTestUtils.createByteFile(mTfs, filePathA, TachyonStorageType.STORE,
+        UnderStorageType.NO_PERSIST, fileASize);
+    Assert.assertTrue(fileExist(new TachyonURI(filePathA)));
+    Assert.assertEquals(fileASize,TachyonBlockStore.get().getUsedBytes());
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: space after comma

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java (hunk 4 above)
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: First file is 20%

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+import tachyon.client.block.TachyonBlockStore;
+import tachyon.shell.AbstractTfsShellTest;
+import tachyon.TachyonURI;
+
+/**
+ * Tests the "getUsedBytes" command
+ */
+public class GetUsedBytesCommandTest extends AbstractTfsShellTest{
+  /**
+   * Tests the available memory on the cluster after adding couple of files
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: available -> used

File: integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    TachyonFile fileA = TachyonFSTestUtils.createByteFile(mTfs, filePathA, TachyonStorageType.STORE,
+        UnderStorageType.NO_PERSIST, fileASize);
+    Assert.assertTrue(fileExist(new TachyonURI(filePathA)));
+    Assert.assertEquals(fileASize,TachyonBlockStore.get().getUsedBytes());
+
+    //Add second file to consume all of tachyon's memory.
+    TachyonFile fileB = TachyonFSTestUtils.createByteFile(mTfs, filePathB, TachyonStorageType.STORE,
+        UnderStorageType.NO_PERSIST, fileBSize);
+    Assert.assertTrue(fileExist(new TachyonURI(filePathB)));
+    Assert.assertEquals(fileASize + fileBSize ,TachyonBlockStore.get().getUsedBytes());
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/shell/command/GetUsedBytesCommandTest.java]: space after comma, not before

**Comment by aaudiber** [comment]
@CruncherBigData Thanks for adding this test, I've left a few comments.

Also, in your PRs please include a link to the relevant JIRA ticket (e.g. https://tachyon.atlassian.net/browse/TACHYON-1376) in the first line of your pull request description (at the top of this page)

**Comment by aaudiber** [comment]
Hey @CruncherBigData, any updates on this?

**Comment by yupeng9** [comment]
@CruncherBigData do you still plan to work on this?

**Comment by aaudiber** [comment]
Closing due to inactivity. @CruncherBigData feel free to reopen if you find time to iterate on the PR.
